includes/apple-exporter/class-exporter.php

```diff
@@ -204,7 +204,7 @@ private function generate_json() {
 		$json = array(
 			'version'    => '1.1',
 			'identifier' => 'post-' . $this->content_id(),
-			'language'   => 'en',
+			'language'   => get_bloginfo( 'language' ),
```

Comment:
This is a good idea, but it needs to be filterable. Can you please add a filter for the language above this block that passes `$this->content_id()` as a parameter? Multilingual sites would be set up with one value for the overall site language with a different language setting on a per-post basis via taxonomy or similar.

Additionally, Apple News only supports English-language content at this time (which is why we had it hardcoded), but I agree that the language property should match the actual language of the site.